ff2mpv.py

```diff
@@ -47,6 +96,22 @@ def get_message():
     return json.loads(message)
 
 
+def get_whitelist():
+    path = get_config_path('whitelist.txt')
+    if path == '':
+        return ['a^'] # impossible regex
+    file = open(path, 'r')
```

Comment:
Nitpick: Use a `with` + iterator construct here, to do automatic cleanup and line I/O:

```python
with open(path) as io:
    return [line.rstrip() for line in io]
```

(Using `rstrip` also avoids chopping a regexp when the last line doesn't end with a newline.)

Reply:
Actually, consider this:

```python
with open(path) as io:
    return [re.compile(line.rstrip()) for line in io]
```

...and see my suggestion below.

Reply:
Done